src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+
+  --link-color: #00ACDC;
 }
+
+.header {
+  width: 100%;
+}
+
+ .wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #FFF;
+}
+
+.logo {
+  margin-left: 50px;
+  display: flex; /* Добавляем flex */
+  align-items: center; /* Центрируем содержимое по вертикали */
+}
+
+.nav_menu {
+  display: flex;
+  margin: 0 50px 0 0;
+  padding: 0;
+}
+
+.nav_menu_item{
+  list-style: none;
+  height: 60px;
+}
+
+.nav_menu_link {
+  font-size: 12px;
+  font-weight: 500;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  position: relative;
+  height: 100%;
+  display: block; /* Делаем ссылку блочным элементом */
+  line-height: 60px; /* Установите line-height равным высоте контейнера */
+}
+
+.nav_menu_link::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+}
+
+a[data-qa="hover"]:hover {
+  color: var(--link-color);
+}
+
+.nav_menu_item:nth-child(n+2) {
+  margin-left: 20px;
+}
+
+.nav_menu_link.is-active {
+  color: var(--link-color); 
+}
+
+.nav_menu_link.is-active::after {
+  background-color: var(--link-color); 
+} 
+
+/* @media screen and (min-width: 1024px) and (max-width: 1200px) {
```

Comment:
pls remove redundant comments

Reply:
Hello @lerastarynets,
I didn't pass all the tests after command git test

Reply:
But I have already found mistake and updated